tests/unittest.py

```diff
@@ -314,11 +314,14 @@ class HomeserverTestCase(TestCase):
         user_id (str): The user ID to assume if auth is hijacked.
         hijack_auth: Whether to hijack auth to return the user specified
         in user_id.
+        lookups: Dictionary of Hostname to IP address (either v4 or v6) that
+        will be fed to the reactor before the homeserver is started.
```

Comment:
Just slightly more readable:

```suggestion
        hijack_auth: Whether to hijack auth to return the user specified
            in user_id.
        lookups: Dictionary of Hostname to IP address (either v4 or v6) that
            will be fed to the reactor before the homeserver is started.
```